gateway/server.go

```diff
@@ -117,6 +119,12 @@ func main() {
 			handlers.DecorateWithBasicAuth(faasHandlers.ListFunctions, credentials)
 		faasHandlers.ScaleFunction =
 			handlers.DecorateWithBasicAuth(faasHandlers.ScaleFunction, credentials)
+		faasHandlers.Alert =
```

Comment:
If I merge this change then the auto-scaling will fail because Prometheus does not send auth strings to the scaling endpoint.